site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            model (str): name to the model
</pre><p> </p><p>Please use Google style docstrings - e.g. <code>model: Model name to use</code>.  This means types are either defined in the function signature (i.e. type hints) or in the right-hand part of the arg doc (e.g. <code>model: Model name as a string</code>)  </p><p> </p><p>Read go/pystyle#docstrings if you can access it, or the <a href="https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings" target="_blank" style="background-color: white; letter-spacing: 0.01em;">external version</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>